src/averageAges.js

```diff
@@ -14,12 +14,28 @@
  *
  * @return {number}
  */
+
+function callbackSum(summa, n) {
+  return (summa + n.died - n.born);
+}
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let peopleNew = [];
+
+  function callbackArr(arr) {
```

Comment:
Consider renaming the function 'callbackArr' to a more descriptive name like 'isPersonFromCentury'.